cfme/tests/configure/test_access_control.py

```diff
@@ -558,6 +562,10 @@ def _test_vm_provision(appliance):
     vms.lcl_btn("Provision VMs")
 
 
+def _test_vm_view(appliance):
+    logger.info("Checking for vms view access")
```

Comment:
**REQUIRED**: ```appliance``` isn't used in this function. Are we requiring it because soon we will need to use vms collections which will require an appliance?

Reply:
It wasn't required, updated to use navigate_to in test instead of external function.